content/scalability/docs/supporting-infrastructure.md

```diff
@@ -0,0 +1,134 @@
+
+We highly recommend customers use Karpenter when possible. It optimizes clusters for cost and scalability and has many features that were built to help customers reduce operational overhead and increase cluster scalability.
+
+## Use many different EC2 instance types
+
+Each AWS region has a limited number of available instances per instance type. If you create a cluster that uses only one instance type and scale the number of nodes beyond the capacity of the region you will receive an error that no instances are available. To avoid this issue you should not arbitrarily limit the type of instances that can be use in your cluster.
+
+Karpenter will use a broad set of compatible instance types by default and will pick an instance at provisioning time based on pending workload requirements, availability, and cost. You can broaden the list of instance types used in the `karpenter.k8s.aws/instance-category` key of [the provisioner](https://karpenter.sh/v0.20.0/concepts/provisioning/).
+
+The Kubernetes Cluster Autoscaler requires node groups to be similarly sized so they can be consistently scaled. You should create multiple groups based on CPU and memory size and scale them independently. Use the [ec2-instance-selector](https://github.com/aws/amazon-ec2-instance-selector) to identify instances that are similarly sized for your node groups.
+
+```
+ec2-instance-selector --service eks --vcpus-min 8 --memory-min 16
+a1.2xlarge
+a1.4xlarge
+a1.metal
+c4.4xlarge
+c4.8xlarge
+c5.12xlarge
+c5.18xlarge
+c5.24xlarge
+c5.2xlarge
+c5.4xlarge
+c5.9xlarge
+c5.metal
+```
+
+## Prefer larger nodes to reduce control plane load
+
+When deciding what instance types to use, fewer large nodes will put less load on the Kubernetes control plane because there will be fewer kubelets and DaemonSets running. However, large nodes may not be utilized the same as smaller nodes depending on your workloads. Node sizes should be evaluated based on your workload availability and scale requirements.
+
+If you run a cluster with three u-24tb1.metal instances (24 TB memory and 448 cores) you would have 3 kublets, but you would be limited to 110 pods per node by default. If your pods use 4 cores each then this might be expected (4 cores x 110 = 440 cores/node). With a 3 node cluster your ability to handle an instance incident would be low. You should specify node requirements and pod spread directly in your workloads so the Kubernetes scheduler can place workloads properly and Karpenter can provision the best node.
```

Comment:
"With a 3 node cluster your ability to handle an instance incident would be low." - can you clarify why?

Reply:
Because if one instance failed it would wipe out 1/3 of your cluster and workloads. Was that not clear from the wording and example?

Reply:
I guessed that was what you are referring to but just adding a sentence or two might clarify that a bit better.